src/overrides.ts

```diff
@@ -47,6 +47,7 @@ class DefaultOverrides {
   // -----------------
   /** a specific seed (default: a random string of 24 characters) */
   readonly SEED_OVERRIDE: string = "";
+  readonly DAILY_RUN_SEED_OVERRIDE: string = "";
```

Comment:
```suggestion
  readonly DAILY_RUN_SEED_OVERRIDE: string | null = null;
```
You used `!isNullOrUndefined()` as the check for activating the override but `""` is neither of those.